non-oss/pharos_pro/addons/kontena-stats/resources/14-kube-state-metrics-deployment.yml.erb

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kube-state-metrics
+  namespace: kontena-stats
+spec:
+  selector:
+    matchLabels:
+      name: kube-state-metrics
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: kube-state-metrics
+    spec:
+      serviceAccountName: kube-state-metrics
+      containers:
+      - name: kube-state-metrics
+        image: <%= image_repository %>/prometheus-kube-state-metrics:v<%= kube_state_metrics_version %>
+        ports:
+        - name: metrics
+          containerPort: 8080
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 5
+          timeoutSeconds: 5
+        resources:
+          requests:
+            cpu: 100m
+            memory: 200Mi
```

Comment:
Does it really need to request `200Mi` of memory?

Reply:
How do you interpret this?https://github.com/kubernetes/kube-state-metrics/blob/master/README.md#resource-recommendation

Reply:
Ok, so it really needs that much... 👀